src/pages/CampusAlumniEndowmentStudioPage.tsx

```
@@ -0,0 +1,218 @@
+              >
+                <X className="w-5 h-5" />
+              </button>
+
+              <div className="mb-6">
+                <h3 className="text-2xl font-black text-white">Propose Alumni Endowment</h3>
+              </div>
+
+              <form onSubmit={handleCreateSubmit} className="space-y-4">
+                <div>
+                  <label className="block text-xs font-bold text-slate-300 mb-1">Fund Name</label>
+                  <input type="text" required value={newFundName} onChange={(e) => setNewFundName(e.target.value)} className="w-full px-3.5 py-2 rounded-xl bg-slate-950 border border-slate-800 text-white text-sm" />
```

ESLint

[error] Replace `·type="text"·required·value={newFundName}·onChange={(e)·=>·setNewFundName(e.target.value)}·className="w-full·px-3.5·py-2·rounded-xl·bg-slate-950·border·border-slate-800·text-white·text-sm"` with `⏎····················type="text"⏎····················required⏎····················value={newFundName}⏎····················onChange={(e)·=>·setNewFundName(e.target.value)}⏎····················className="w-full·px-3.5·py-2·rounded-xl·bg-slate-950·border·border-slate-800·text-white·text-sm"⏎·················`

(prettier/prettier)